flood_adapt/object_model/direct_impacts.py

```diff
@@ -174,12 +182,49 @@ def run_fiat(self):
             else:
                 print("Impact measure type not recognized!")
 
+        # fa.fiat_model.setup_social_vulnerability()
         # Set FIAT hazard
-        # fa.set_hazard(self.hazard)
+        map_fn = self.get_sfincs_map_path(self.hazard.sfincs_map_path)
+
+        is_risk = self.hazard.event_mode == Mode.risk
+        var = "zsmax" if self.hazard.event_mode == Mode.risk else "risk_maps"
+
+        fa.fiat_model.setup_hazard(
+            map_fn=map_fn,
+            map_type="water_depth",
+            rp=None,  # change this in new version
+            crs=None,  # change this in new version
+            nodata=-999,  # change this in new version
+            var=var,
+            chunks="auto",
+            name_catalog=None,
+            risk_output=is_risk,
+        )
```

Comment:
I was expecting that this should be in the fiat_adapter in the set_hazard method

Reply:
Ok I indeed think that this need some refactoring, but I do not think that it should be in the set_hazard method, because set_hazard and setup_hazard do different things. I refactored this part and made a new method